core/class-shoutem-api-encryption.php

```diff
@@ -45,41 +43,10 @@ function decrypt( $data, $key ) {
 	}
 }
 
-class MyCryptEncryptor {
-
-	function __construct() {
-	}
-
-	function encrypt( $text, $key ) {
-		 $data = mcrypt_encrypt( MCRYPT_RIJNDAEL_128, $key, $text, MCRYPT_MODE_ECB, $this->get_iv() );
-		 return base64_encode( $data );
-	}
-
-	function decrypt( $data, $key ) {
-		$data = base64_decode( $data );
-		return mcrypt_decrypt( MCRYPT_RIJNDAEL_128, $key, $data, MCRYPT_MODE_ECB, $this->get_iv() );
-	}
-
-	function get_iv() {
-		if ( isset( $this->iv ) == false ) {
-			$ivs = mcrypt_get_iv_size( MCRYPT_RIJNDAEL_128, MCRYPT_MODE_ECB );
-			$this->iv = mcrypt_create_iv( $ivs );
-		}
-		return $this->iv;
-	}
-}
-
 class ShoutemApiEncryption {
-
 	function __construct( $key ) {
 		$this->key = $key;
-
-		if ( function_exists( 'mcrypt_list_algorithms' ) ) {
-			$this->encryptor = new MyCryptEncryptor();
-		} else {
```

Comment:
Since `mcrypt_list_algorithms` function has been deprecated as of PHP 7.1.0 and removed as of PHP 7.2.0, this piece of code is literally dead. Removing it solves the issue.